.stylelintrc.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I believe this file should be in the client folder. The backend guys should not care about this

Reply:
The sylelint cli command has an option to pass in the path of the config file, fixing this won't be a problem will do it right away.